ansible/README.md

```diff
@@ -25,7 +25,7 @@ brew install python
 pip install ansible==2.0.2.0
 
 cd ansible
-ansible-playbook setup.yml [-e docker_machine_name=whisk]
+ansible-playbook -i environments/mac setup.yml [-e docker_machine_name=whisk]
```

Comment:
@tuler : I'd love to support it, if I could use it :). Until now, no reaction from docker for trying the new docker for mac.